server/entity/animation/animation.go

```diff
@@ -0,0 +1,61 @@
+package animation
+
+// Animation represents an animation & controller that may be attached to an entity.
+// Animations and controllers must be defined in a resource pack
```

Comment:
Something like this would sound better
```go
// Animation represents an animation that may be played on an entity from an active resource pack on
// the client.
```